src/main/flight/navigation_rewrite_multicopter.c

```diff
@@ -481,13 +481,22 @@ bool isMulticopterLandingDetected(uint32_t * landingTimer, bool * hasHadSomeVelo
     // check if we are moving horizontally
     bool horizontalMovement = sqrtf(sq(posControl.actualState.vel.V.X) + sq(posControl.actualState.vel.V.Y)) > 100.0f;
 
-    // Throttle should be low enough
+    // We have likely landed if throttle is 40 units below average descend throttle
     // We use rcCommandAdjustedThrottle to keep track of NAV corrected throttle (isLandingDetected is executed
     // from processRx() and rcCommand at that moment holds rc input, not adjusted values from NAV core)
-    bool minimalThrust = rcCommandAdjustedThrottle < posControl.navConfig->mc_min_fly_throttle;
-    
+    *landingThrSamples += 1;
+    *landingThrSum += rcCommandAdjustedThrottle;
+    bool minimalThrust= rcCommandAdjustedThrottle < *landingThrSum / *landingThrSamples - 40;
```

Comment:
Renamed minimalThrust to isAtMinimalThrust.

Hmm, let me see if i understand your percentage thingy right:
My minthrottle=1040, hover=1220, so you formula would give 1220-1040=180, lets say current throttle is at 1180-1040=140, 140/180=77%, was it something like this you meant? That would require a bigger change in absolute throttle value with higher hover though, not sure if we want that...?